app/src/main/java/com/android/sample/model/image/ImageRepositoryFirestore.kt

```diff
@@ -140,4 +140,17 @@ constructor(private val firestore: FirebaseFirestore, private val storage: Fireb
         },
         onFailure)
   }
```

Comment:
Added a new method deleteProfilePicture to delete a profile picture from Firebase storage.